docs/DeveloperGuide.adoc

```diff
@@ -277,90 +642,247 @@ Refer to the guide <<DevOps#, here>>.
+*Value proposition*: TagLine manages notes faster than a typical mouse/GUI driven app
 
 [appendix]
 == User Stories
 
 Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (unlikely to have) - `*`
 
-[width="59%",cols="22%,<23%,<25%,<30%",options="header",]
+[width="90%",cols="15%,<20%,<30%,<35%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|`* * *` |user |add a new contact |
+|`* * *` |user |edit a contact |update outdated information
+|`* * *` |user |delete a contact |remove entries that I no longer need
+|`* * *` |user |find a contact by name |locate details of contacts without having to go through the entire list
+|`* * *` |user |group my contacts |manage contacts for different occasions better
+|`* * *` |user |view all contacts in a group |
+|`* * *` |user |add new notes |
+|`* * *` |user |edit a note |fix typos or incorrect details
+|`* * *` |user |delete a note |clean up my app
+|`* * *` |user |list all notes by chronological order |view most relevant notes first
+|`* * *` |user |tag my notes |group related notes together
+|`* * *` |user |view all notes according to tags |view only notes related to an issue
+|`* * *` |user |view all notes related to a contact |discuss these notes with them when I meet them
+|`* * *` |user |view all notes related to a group |
+|`* *` |user |view all notes related to groups as well when querying for a person |view all information associated with that person at a glance
+|`* *` |user with many friends with the same name |be able to differentiate them easily |locate a specific person
+|`* *` |user |archive old notes |keep them while not cluttering my app page
+|`* *` |user |export all my data and create a backup |keep my data somewhere safe
+|`* *` |new user |get suggestions when typing commands |do not need to memorize commands
+|`*` |user |embed links in my notes |directly access relevant webpages
+|`*` |user |associate photos with notes |store and view related photos and notes together
+|`*` |user |add text styles |personalize my entries
+|`*` |user |colour entries with the same tag |organize my notes better
+|`*` |user |prompted for correction when I make typos |fix my command without re-typing it entirely
+|`*` |user |prompted for confirmation when I delete or edit notes or contacts |avoid making irreversible mistakes
+|`*` |user |lock notes with authentication |keep my notes secure
+|=======================================================================
 
-|`* * *` |user |add a new person |
+[appendix]
+== Use Cases
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+(For each of the use cases below, the *System* is `TagLine` and the *Actor* is the `user`, unless specified otherwise)
```

Comment:
It would be better to refer `System` instead of `TaglLine` in the use cases so that first-time readers will not get confused when they are reading the uses cases.